src/lib/ui/next/customNextPage.js

```diff
@@ -1,17 +1,17 @@
-export default `
+export default `'use client';
 import Head from 'next/head';
 import Image from 'next/image';
 import { useEffect } from 'react';
 import GradientBG from '../components/GradientBG.js';
 import styles from '../styles/Home.module.css';
-import heroMinaLogo from '../../public/assets/hero-mina-logo.svg';
-import arrowRightSmall from '../../public/assets/arrow-right-small.svg';
+import heroMinaLogo from '../public/assets/hero-mina-logo.svg';
+import arrowRightSmall from '../public/assets/arrow-right-small.svg';
 
 export default function Home() {
   useEffect(() => {
     (async () => {
       const { Mina, PublicKey } = await import('o1js');
-      const { Add } = await import('../../../contracts/build/src/');
+      const { Add } = await import('../../contracts/build/src/');
 
       // Update this to use the address (public key) for your zkApp account.
       // To try it out, you can try this address for an example "Add" smart contract that we've deployed to
```

Comment:
Did you test if uncommenting this code works with your changes?

Reply:
The code doesn't error, but it's also not really doing anything. Looking at this more closely it seems like that zkapp address might be out of date. Maybe we should either remove the example code or update it to really interact with the network

```ts
const { Mina, PublicKey,fetchAccount } = await import('o1js');  
const { Add } = await import('../../contracts/build/src/');  
// connect Mina to testnet
const Network = Mina.Network(  
    'https://api.minascan.io/node/devnet/v1/graphql'  
);  
Mina.setActiveInstance(Network);  
// address of deployed Add contract on testnet https://minascan.io/devnet/account/B62qnTDEeYtBHBePA4yhCt4TCgDtA4L2CGvK7PirbJyX4pKH8bmtWe5
const zkAppAddress = `B62qnTDEeYtBHBePA4yhCt4TCgDtA4L2CGvK7PirbJyX4pKH8bmtWe5`;  
await fetchAccount({publicKey: zkAppAddress});  
const zkApp = new Add(PublicKey.fromBase58(zkAppAddress));  
console.log("Current num: ", zkApp.num.get().toString());
```

Reply:
Yeah, sounds like a different PR though 👍

We can reconsider what the right format is for demoing o1js in a boilerplate app.